<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FitView</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="centralwidget2">
      <widget class="QWidget" name="mainpage" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="midLineWidth">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="opaqueResize">
           <bool>false</bool>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QSplitter" name="splitter_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="opaqueResize">
               <bool>false</bool>
              </property>
              <property name="childrenCollapsible">
               <bool>false</bool>
              </property>
              <widget class="QWidget" name="metadataFrame" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>250</width>
                 <height>250</height>
                </size>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="sizeConstraint">
                 <enum>QLayout::SetDefaultConstraint</enum>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QStackedWidget" name="metadataStackedWidget">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Plain</enum>
                  </property>
                  <property name="currentIndex">
                   <number>1</number>
                  </property>
                  <widget class="QWidget" name="page"/>
                  <widget class="QWidget" name="page_2"/>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QFrame" name="graph">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QTableWidget" name="all_events_table">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <column>
            <property name="text">
             <string>Event Date</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Event Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Event Type</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Event Subtype</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>RowID</string>
            </property>
           </column>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="swimpage"/>
      <widget class="QWidget" name="runpage"/>
      <widget class="QWidget" name="cyclingpage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionNew_Logbook"/>
   <addaction name="actionOpen_Logbook"/>
   <addaction name="separator"/>
   <addaction name="actionImport_File"/>
   <addaction name="actionSync_with_Garmin"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Logbook"/>
    <addaction name="actionOpen_Logbook"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuTest"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/menu/img/No.svg</normaloff>:/menu/img/No.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionOpen_Logbook">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/menu/img/folder_file_open.svg</normaloff>:/menu/img/folder_file_open.svg</iconset>
   </property>
   <property name="text">
    <string>Open Logbook</string>
   </property>
   <property name="toolTip">
    <string>Open Logbook</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew_Logbook">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/menu/img/file_blanc_star.svg</normaloff>:/menu/img/file_blanc_star.svg</iconset>
   </property>
   <property name="text">
    <string>New Logbook</string>
   </property>
   <property name="toolTip">
    <string>New Logbook</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionImport_File">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/menu/img/database_import.svg</normaloff>:/menu/img/database_import.svg</iconset>
   </property>
   <property name="text">
    <string>Import File</string>
   </property>
   <property name="toolTip">
    <string>Import File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSync_with_Garmin">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/menu/img/import-export.svg</normaloff>:/menu/img/import-export.svg</iconset>
   </property>
   <property name="text">
    <string>Sync with Garmin</string>
   </property>
   <property name="toolTip">
    <string>Sync with Garmin</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
